extensions/CHCAT1320/midi.js

```diff
@@ -0,0 +1,111 @@
+// Name:MIDI Parser
+// ID: midiParser
+// Description: Parse MIDI files into JSON.
+// By: CHCAT1320
+// License: MIT
+
+
+//  tonejs/midi 
```

Comment:
This should probably be a full comment block including link to the original code and maybe license text as well - check other extensions for examples